docs/guides/headless-cms/schedule-content/index.md

```diff
@@ -1,13 +1,17 @@
 # Scheduling Future Content
 
-<Card 
-  title="Dynamic Websites" 
+<Card
+  title="Dynamic Websites"
   h="2"
-  text="Learn how to conditionally show content on dynamic websites." 
-  url="/guides/headless-cms/schedule-content/dynamic-sites" />
+  text="Learn how to conditionally show content on dynamic websites."
+  url="/guides/headless-cms/schedule-content/dynamic-sites"
+  add-margin
```

Comment:
The `addMargin` prop was removed in https://github.com/directus/directus/pull/18911/files#diff-7f7a2886ba028429230a8f3655f1c8b51e6b62ee8b08f14d14bbe7745a22f40eL16